src/components/Help/Steppers/UserMountsStepper.js

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import {
+    Stepper,
+    Step,
+    StepLabel,
+    Typography,
+    Button,
+} from '@material-ui/core';
+import { USER_MOUNTS_STEPPER_STEPS, USER_MOUNTS_STEPPER_INSTRUCTIONS } from 'utils/constants';
+import 'components/Help/Steppers/Stepper.scss';
+import UserMountsInstructionsSelectUser from 'utils/Assets/UserMountsInstructionsSelectUser.gif';
+import UserMountsInstructionsChangeOwned from 'utils/Assets/UserMountsInstructionsChangeOwned.gif';
+import UserMountsInstructionsConfirmChanges from 'utils/Assets/UserMountsInstructionsConfirmChanges.gif';
+import UserMountsInstructionsExpansionFilter from 'utils/Assets/UserMountsInstructionsExpansionFilter.gif';
+
+function getStepContent(stepIndex) {
+    switch (stepIndex) {
+        case 0:
+        return (
+            <div>
+                <Typography>{USER_MOUNTS_STEPPER_INSTRUCTIONS[0]}</Typography>
+                <div className='box'><img src={UserMountsInstructionsSelectUser} alt="How to select."/></div>
+            </div>
+            
+        );
+        case 1:
+        return (
+            <div>
+                <Typography>{USER_MOUNTS_STEPPER_INSTRUCTIONS[1]}</Typography>
+                <div className='box'><img src={UserMountsInstructionsChangeOwned} alt="How to edit."/></div>
+            </div>
+        );
+        case 2:
+        return (
+            <div>
+                <Typography>{USER_MOUNTS_STEPPER_INSTRUCTIONS[2]}</Typography>
+                <div className='box'><img src={UserMountsInstructionsConfirmChanges} alt="How to edit."/></div>
+            </div>
+        );
+        case 3:
+        return (
+            <div>
+                <Typography>{USER_MOUNTS_STEPPER_INSTRUCTIONS[3]}</Typography>
+                <div className='box'><img src={UserMountsInstructionsExpansionFilter} alt="How to change expansion."/></div>
+            </div>
+        );
+        case 4:
+        return (
+            <div>
+                <Typography>{USER_MOUNTS_STEPPER_INSTRUCTIONS[4]}</Typography>
+                <div className='box'><img src={UserMountsInstructionsExpansionFilter} alt="How to search."/></div>
+            </div>
+        );
+        default:
+        return 'Unknown stepIndex';
+    }
+}
```

Comment:
This is a lot of hard-coded content. I would recommend declaring a constant with a similar format as the following code snippet:
Example:
```
export const USER_MOUNTS_STEPPER_INSTRUCTIONS = {
  0: {
       instructionText: "How to select",
       image: "utils/Assets/UserMountsInstructionsSelectUser.gif" 
  },
  ...
}
```

Reply:
Design implemented